ts/Series/ColumnRangeSeries.ts

```diff
@@ -130,6 +134,13 @@ BaseSeries.seriesType<typeof Highcharts.ColumnRangeSeries>('columnrange', 'arear
     (defaultOptions.plotOptions as any).arearange,
     columnRangeOptions
 ), {
+    setOptions: function (this: Highcharts.ColumnRangeSeries): Highcharts.ColumnRangeSeriesOptions {
+        var options = arearangeProto.setOptions.apply(this, arguments);
+        if (options.stacking) {
```

Comment:
Add a little comment with a reference to the issue number, so that future us can understand why we're doing this.